pkgs/development/python-modules/sqlcipher3/default.nix

```diff
@@ -0,0 +1,39 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchPypi,
+  setuptools,
+  sqlcipher,
+  openssl,
+}:
+let
+  pname = "sqlcipher3";
+  version = "0.5.4";
+in
+buildPythonPackage {
+  inherit pname version;
+  pyproject = true;
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-4w/1jWTdQ+Gezt3RARahonrR2YiMxCRcdfK9qbA4Tnc=";
+  };
+
+  nativeBuildInputs = [
+    setuptools
+  ];
+
+  buildInputs = [
+    sqlcipher
+    openssl
+  ];
+
```

Comment:
Could you enable tests with `pythonImportsCheck` and `pytestCheckHook`?

https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#testing-python-packages-testing-python-packages

Reply:
@phaer that was your code, would you mind to edit this? (Push to belamu:353082-bump-signal-export-to-3.2.0 should be possible for you)

Reply:
see below: pythonImportCheck done. pytestCheckHook not really feasible.